### PR TITLE
Biblio contrbuter field collection

### PR DESCRIPTION
**Pull request by RoySegall**
Adding field condition and order the biblio example module.

### CODE REVIEW

File: modules/biblio_example/biblio_example.module [hunk 1]
```diff
@@ -92,4 +141,60 @@ function biblio_example_init() {
...
   $wrapper->save();
+
+  return '';
 }
+
+/**
+ * Create a field collection field for the biblio contributor.
+ */
+function biblio_example_create_field_collection_fields() {
+  if (!module_exists('field_collection')) {
```
Comment (amitaibu) [inline modules/biblio_example/biblio_example.module]: Lets make field_collection dependency.

Comment (amitaibu) [inline modules/biblio_example/biblio_example.module]: After that, you can merge.

File: biblio.module [hunk 2]
```diff
@@ -905,5 +912,46 @@ function biblio_biblio_fields_info() {
     ),
   );
 
+  $items['biblio_contributor_reference'] = array(
```
Comment (amitaibu) [inline biblio.module]: Remove the word reference, it's enough `biblio_contributor`

**Comment by amitaibu** [comment]
Actually don't commit yet,  I want to do some changes.

**Comment by RoySegall** [comment]
Merged.

**Pull request merged**